src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -281,6 +297,10 @@ export default class CardanoSignTransaction extends AbstractMethod {
         for (const withdrawal of this.params.withdrawals) {
             await typedCall('CardanoTxWithdrawal', 'CardanoTxItemAck', withdrawal);
         }
+        // collateral
+        for (const { input } of this.params.collateralInputsWithPath) {
```

Comment:
I'd call the variable `collateralInput` instead of just `input`.

Reply:
This is perhaps forgotten? Or you decided otherwise?

Reply:
This exact occurence of `input` is here due to the `InputWithPath` type (note the curly braces). I changed all `input` variable occurences to `collateralInput`, but this has to stay the same unless we want to create a `CollateralInputWithPath` type.

Reply:
True, of course, I see. I didn't realise that. Alright, let's leave it like this, sorry. Another alternative would be to use `const { input: collateralInput } of ...`, but I guess it's clear what's happening.